src/main/java/org/fundacion/jala/converter/models/parameter/PythonEnum.java

```diff
@@ -5,12 +5,14 @@
  * ("Confidential Information"). You shall not disclose such Confidential
  * Information and shall use it only in accordance with the terms of the
  * license agreement you entered into with Fundacion Jala
- */
-/**
+ *
  * @author Joel Rodrigo Rojas Roman
  */
 package org.fundacion.jala.converter.models.parameter;
 
+/**
+ * This class refers to binary file to python.
```

Comment:
Binary files for Python, could be a better description.

Reply:
ok